audio/modplug-decoding/Makefile

```diff
@@ -50,14 +50,14 @@ CFLAGS	:=	-g -Wall -O2 -mword-relocations \
 			-ffunction-sections \
 			$(ARCH)
 
-CFLAGS	+=	$(INCLUDE) -DARM11 -D_3DS
+CFLAGS	+=	$(INCLUDE) -DARM11 -D__3DS__
 
 CXXFLAGS	:= $(CFLAGS) -fno-rtti -fno-exceptions -std=gnu++11
 
 ASFLAGS	:=	-g $(ARCH)
 LDFLAGS	=	-specs=3dsx.specs -g $(ARCH) -Wl,-Map,$(notdir $*.map)
 
-LIBS	:= `$(PREFIX)pkg-config --libs libmodplug` -lz -lctru -lm
+LIBS	:= `$(PREFIX)pkg-config --libs libmodplug` -lctru -lm
```

Comment:
I suppose this is intentional?

Reply:
Yes. `-lz` isn't needed for that example at all afaik. Am I missing something?

Reply:
Just checking. It builds, right?

Reply:
Yes.